src/OrganizationIntegration/OrganizationIntegrationView/SchedulingView/SchedulingView.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  Checkbox,
+  Col,
+  KeyValue,
+  Row,
+} from '@folio/stripes/components';
+
+import {
+  SCHEDULE_PERIODS,
+  WEEKDAYS,
+} from '../../constants';
+
+const SchedulingView = ({ ediSchedule = {} }) => {
+  const isScheduleEnabled = ediSchedule.enableScheduledExport;
+  const schedulePeriod = ediSchedule.scheduleParameters?.schedulePeriod;
+
+  return (
+    <Accordion
+      id="scheduling"
+      label={<FormattedMessage id="ui-organizations.integration.scheduling" />}
+    >
+      <Row>
+        <Col xs={3}>
+          <Checkbox
+            label={<FormattedMessage id="ui-organizations.integration.scheduling.scheduleEDI" />}
+            checked={ediSchedule.enableScheduledExport}
+            vertical
+            disabled
+          />
+        </Col>
+
+        {isScheduleEnabled && (
+          <>
+            <Col xs={3}>
+              <KeyValue
+                label={<FormattedMessage id="ui-organizations.integration.scheduling.schedulePeriod" />}
+                value={SCHEDULE_PERIODS[ediSchedule.scheduleParameters?.schedulePeriod]}
```

Comment:
I think, that here you try to get period by value, not by key: 
`ediSchedule.scheduleParameters?.schedulePeriod -> 'HOUR' | 'DAY' ...`
`SCHEDULE_PERIODS: {  hours: 'HOUR' ... }`

Reply:
Good catch, thx